### PR TITLE
updated policies page with the computer policy

### PR DESCRIPTION
**Pull request by nickyvm**
I updated this page with the computer policy link

### CODE REVIEW

File: content/company-info-and-process/policies/index.md [hunk 1]
```diff
@@ -1,18 +1,21 @@
 # Company wide directives and policies
 
-## Teammate privacy and trust
+## Directives
```
Comment (nicksnyder) [inline content/company-info-and-process/policies/index.md]: what is the difference between a directive and a policy?

Comment (nickyvm) [reply content/company-info-and-process/policies/index.md]: It isn't part of our SOC 2 policies so I felt weird adding it below. You and Dora suggested I add it here but doesn't seem like an official policy document. 

I see directives as the specifics that are shaped by the policy. Ie the computer requirements are shaped by the asset management policy but aren't explicitly stated there. I think eventually they will be merged as the 'policies' need to be re-written in Sourcegraph languange.

Comment (nicksnyder) [reply content/company-info-and-process/policies/index.md]: Maybe I am missing something and "policy" is a term of art that means something special in compliance land, but I would casually call our device requirements a "policy" and would find it less confusion if they were just all grouped together as such.

Comment (nicksnyder) [reply content/company-info-and-process/policies/index.md:3]: Maybe I am missing something and "policy" is a term of art that means something special in compliance land, but I would casually call our device requirements a "policy" and would find it less confusion if they were just all grouped together as such.

Comment (nicksnyder) [reply content/company-info-and-process/policies/index.md]: I don't feel strongly about this. I defer to you on what you think makes sense here (either leaving it separate or not adding it to this list at all)

**Review by nicksnyder — APPROVED** [review]

**Review by nickyvm — COMMENTED** [review]
I agree it was wordy and unnecessary. I've added it to the list

**Pull request merged**